### DISCUSSION TITLE
Installing SignalK plugins from command line

### DISCUSSION THREAD

**Discussion by toastygm** [bareboat-necessities/lysmarine_gen#352] [Q&A] [answered]
Since the recommended way of upgrading BBN OS is to use a new image, I want to automate my installation as much as possible.  How do I install new plugins for SignalK from the command line?

**Comment by mgrouch** [comment, marked as answer]
You can see how it’s done here

https://github.com/bareboat-necessities/lysmarine_gen/blob/bullseye/install-scripts/4-server/04-signalk.sh
